lib/prelude/sprite/mod.dart

```diff
@@ -0,0 +1,9 @@
+import 'package:backbone/builders.dart';
+import 'package:backbone/prelude/sprite/system.dart';
+import 'package:backbone/prelude/sprite/trait.dart';
+
+/// Add the spritetrait and spriteSystem
```

Comment:
```suggestion
```

I think it's a kind of redundant comment :D

Reply:
This is a document comment, so people just hovering over the name in code always see what's inside without opening the file. I guess this doesnt hurt